### ISSUE TITLE
Old versions of PowerShell show ugly message (before the nice message) when running flutter

### ISSUE THREAD

**Issue by CulionBEAR** [flutter/flutter#28699]
I put "D:\FFlutter\flutter\bin" in the Path,but there is  a problem when I input "flutter doctor"!
What shoulde I do?

D:\FFlutter\flutter>flutter doctor
Checking Dart SDK version...
无法将“Unblock-File”项识别为 cmdlet、函数、脚本文件或可运行程序的名称。请检查
名称的拼写，如果包括路径，请确保路径正确，然后重试。
所在位置 行:1 字符: 13
+ Unblock-File <<<<  -Path 'D:\FFlutter\flutter/bin/internal/update_dart_sdk.ps
1'; & 'D:\FFlutter\flutter/bin/internal/update_dart_sdk.ps1'
    + CategoryInfo          : ObjectNotFound: (Unblock-File:String) [], Comman
   dNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Flutter requires PowerShell 5.0 or newer.
See https://flutter.io/docs/get-started/install/windows for more.
Building flutter tool...

**Comment by CulionBEAR** [comment]
I solved the problem！

**Comment by kubilaykilinc** [comment]
I have same error. How can you solved?

**Comment by jonahwilliams** [comment]
Duplicate of https://github.com/flutter/flutter/issues/27938
